Lib/test/test_urllib.py

```diff
@@ -1201,6 +1199,20 @@ def test_unquote_with_unicode(self):
         self.assertEqual(expect, result,
                          "using unquote(): %r != %r" % (expect, result))
 
+    def test_unquoting_with_bytes_input(self):
+        given = b'bl\xc3\xa5b\xc3\xa6rsyltet\xc3\xb8y'
```

Comment:
Please add a simpler test case here in addition to the others. Also, please add a (short) comment explaining what each of the more complex test cases is checking.